packages/frontend/src/components/dialogs/EditAccountAndPasswordDialog.tsx

```diff
@@ -50,30 +56,26 @@ function EditAccountInner(onClose: DialogProps['onClose']) {
     if (window.__selectedAccountId === undefined) {
       throw new Error('can not load settings when no account is selected')
     }
-    const accountSettings: Credentials =
-      (await BackendRemote.rpc.batchGetConfig(window.__selectedAccountId, [
-        'addr',
-        'mail_pw',
-        'sentbox_watch',
-        'mvbox_move',
-        'only_fetch_mvbox',
-        'e2ee_enabled',
-        'mail_server',
-        'mail_user',
-        'mail_port',
-        'mail_security',
-        'imap_certificate_checks',
-        'send_user',
-        'send_pw',
-        'send_server',
-        'send_port',
-        'send_security',
-        'smtp_certificate_checks',
-        'proxy_enabled',
-        'proxy_url',
-      ])) as unknown as Credentials
-    setInitialAccountSettings(accountSettings)
-    _setAccountSettings(accountSettings)
+    const accountId = window.__selectedAccountId
+    const transports = await BackendRemote.rpc.listTransports(accountId)
+    if (transports.length === 0) {
+      throw new Error('no transport found')
+    }
+    const accountSettings: T.EnteredLoginParam = transports[0]
```

Comment:
Is this what we're supposed to do? Just take the first transport? @Hocuri

Reply:
For now yes, since there are no multiple transports yet. We have to adapt this (and more things) before we have multi transport of course